crt_portal/cts_forms/templates/forms/complaint_view/dashboard/filter-controls.html

```diff
@@ -13,7 +13,7 @@ <h2 class="intake-section-title">Team Management</h2>
     </div>
 
     <div class="text-right">
-      <button class="usa-button" type="submit" id="apply-filters-button">
+      <button class="usa-button" type="submit" id="apply-filters-button" onclick="gtag('event', 'click', {'event_name': 'dashboard filter'});">
```

Comment:
Is it possible to confirm that these work in local/dev/staging?

Either a separate GA property, or something simple like rigging gtag to console.log the events and params we're sending along?